helm/application-namespace-prep/tests/conjur-connect-configmap_test.yaml

```diff
@@ -0,0 +1,72 @@
+suite: test conjur-connect_configmap
+
+templates:
+  - conjur-connect-configmap.yaml
+
+defaults: &defaultRequired
+  authnK8s.goldenConfigMap: authn-k8s-configmap
+  authnK8s.namespace: golden
+  
+
+tests:
+  #=======================================================================
+  - it: should not create a ConfigMap if ConfigMap creation is disabled
+  #=======================================================================
+    set:
+      conjurConfigMap.create: false
+    asserts:
+      - hasDocuments:
+          count: 0
+
+  #=======================================================================
+  - it: should use default values when those values are not set explicitly
+  #=======================================================================
+    set:
+      # Set required values
+      <<: *defaultRequired
+      test.mock.enable: true
+
+
+    asserts:
+      # Confirm that a ConfigMap has been created
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+
+
+      # Confirm that default values have been used
+      - equal:
+          path: metadata.name
+          value: conjur-connect-configmap
+
+
+     # Confirm that values are read from lookup()
+
+      - equal:
+          path: data.CONJUR_ACCOUNT
+          value: "mock-conjur-account"
+
+      #- equal:
```

Comment:
Is this supposed to be commented out, or left over from testing?

Reply:
left over from testing - removed it